test/BeanstalkClientConnectionClosedTest.php

```diff
@@ -34,20 +32,21 @@ public function tearDown() {
      * @param $reserveTimeout int|null Seconds
      * @param $connectionCloseTimeout int Milliseconds
      * @param $testFailTimeout int Milliseconds
-     * @return \Generator
      */
-    public function testReserve($reserveTimeout, $connectionCloseTimeout, $testFailTimeout) {
+    public function testReserve(?int $reserveTimeout, int $connectionCloseTimeout, int $testFailTimeout): void {
```

Comment:
For some reason this is always failing on the first invocation with no exception thrown, but passing (i.e. throwing correctly) on the second invocation